### ISSUE TITLE
reorganize if file.exists logic in set_physical for size, authentication

### ISSUE THREAD

**Issue by scelmendorf** [cboettig/eml2#17]
set_physical doesn't seem to work correctly when the parameters are supplied rather than derived via the set_physical code. Specifically, I think the 

```r
authentication <- list(authentication = authentication, method = authMethod)
```

and 

```r
size <- list(size = size, unit = sizeUnit)
```

lines need to be moved OUTSIDE of the if(file.exists... loop, otherwise the resulting object doesn't have the sizeUnit or authMethod elements

**Comment by cboettig** [comment]
thanks! haven't had a chance to look at this yet but that sounds right.  Want to send a PR? Otherwise I'll get around to this later this week.

**Comment by scelmendorf** [comment]
Sure! Also traveling now but will when I'm back.

**Comment by cboettig** [comment]
Closed by #19
